suites/build.gradle.kts

```diff
@@ -43,7 +43,7 @@ tasks.register<Test>("runSuites") {
     group = "suites"
     modularity.inferModulePath = false
 
-    // @todo(#343) - :server:createProductionDotEnv should disappear
+    // @todo(#343) - :server:createDotEnv should disappear
```

Comment:
```suggestion
    // @todo(#343) - :server:createProductionDotEnv should disappear
```

Reply:
@georgi-l95 feel free to close this comment, it is non blocking, up to you.

Reply:
Thanks for catching it, renamed.